frontend/appflowy_flutter/lib/mobile/presentation/database/view/database_field_list.dart

```diff
@@ -61,6 +61,10 @@ class _MobileDatabaseFieldListBody extends StatelessWidget {
           if (state.fieldContexts.isEmpty) {
             return const SizedBox.shrink();
           }
+
+          final mediaQuery = MediaQuery.of(context);
+          final bottomPadding = mediaQuery.viewPadding.bottom +
```

Comment:
maybe we should add this padding calculation to an extension, so that we can reuse it in the mobile bottom sheet.

Reply:
Sure, sounds good. Maybe `context.bottomSheetPadding`? But just a heads up, I have to put it here rather than in the bottom sheet function because ListView here doesn't play well with the column there.

Reply:
`context.bottomSheetPadding` is ok.